posthog/user_scripts/v5/get_survey_response.py

```diff
@@ -0,0 +1,121 @@
+#!/usr/bin/env python3
```

Comment:
we already have this file here https://github.com/PostHog/posthog/pull/30182/files#diff-fffa98dc304d8240297bc8824c2dd21c12605991a6bc155197a33340975c4c2d
under the PR https://github.com/PostHog/posthog/pull/30182
so I wonder why do we have 2 times? do we need them twice? how do we make sure they are in sync?